src/Controller/UserController.php

```diff
@@ -77,6 +79,7 @@ public function newAction(Request $request, UserManagerInterface $userManager, E
      * Displays a form to edit an existing User entity.
      *
      * @Route("/users/{id}/edit", name="user_edit", methods={"GET", "POST"})
+     * @IsGranted("EDIT", subject="user")
```

Comment:
The `user` `subject` means it has to use the `UserVoter`?

Reply:
kind of yes! it will use the `UserVoter` because it will be the one supporting a `User` entity as subject